src/prog/cfg_AbstractCFGBuilder.cpp

```diff
@@ -311,6 +311,9 @@ void AbstractCFGBuilder::buildEdges(CFGMaker& m) {
 						ts.clear();
 						targets(i, ts, workspace(), otawa::BRANCH_TARGET);
 
+						// clear NO_BLOCK marked targets
+						// TODO JRU
```

Comment:
Will you address this during this PR?

Reply:
No, this PR only adds syntax support. I am not definite yet on whether we want the CFGTransformer pass that uses the NO_BLOCK (that is already implemented and tested within rocqstat) or to implement `noblock` functional support during the CFGBuilder phase. The second one is more intuitive (especially because it's weird to modify OTAWA with syntax that can only be used with a Rocqstat plugin), so I suppose so I will give it a short try, but if it procs any issue I'll revert to the first approach with is pretty much a oneliner.